### PR TITLE
#212: Change project name to Jakarta Persistence

### LINKED ISSUE #212: Change project name to Jakarta Persistence
Background information:
https://waynebeaton.wordpress.com/2019/04/04/renaming-java-ee-specifications-for-jakarta-ee/

Checklist:
- [ ]  GitHub
- [ ]  README file
- [ ]  CONTRIBUTING file
- [x]  Eclipse Project under https://projects.eclipse.org/projects/ee4j

### PR DESCRIPTION
**Pull request by lukasj**
Fixes #212 
Signed-off-by: Lukas Jungmann <lukas.jungmann@oracle.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
-# Contributing to Eclipse Project for JPA
+# Contributing to Eclipse Jakarta Persistence
```
Comment (bshannon) [inline CONTRIBUTING.md]: You can remove "Eclipse" here.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 
 Thanks for your interest in this project.
 
 ## Project description
 
 The Java Persistence API is the Java API for the management of persistence and
-object/relational mapping in Jakarta EE and Java SE environments.
+object/relational mapping in Jakarta EE and Java SE environments.
```
Comment (bshannon) [inline CONTRIBUTING.md]: Did you add a space at the end of this line?

File: NOTICE.md [hunk 3]
```diff
@@ -1,12 +1,13 @@
-# Notices for Eclipse Project for JPA
+# Notices for Eclipse Jakarta Persistence
```
Comment (bshannon) [inline NOTICE.md]: Again, you can remove "Eclipse".

File: CONTRIBUTING.md (hunk 1 above)
Comment (lukasj) [reply CONTRIBUTING.md]: done

File: CONTRIBUTING.md (hunk 2 above)
Comment (lukasj) [reply CONTRIBUTING.md]: done

File: NOTICE.md (hunk 3 above)
Comment (lukasj) [reply NOTICE.md]: done

**Pull request merged**